core/src/main/java/com/linecorp/armeria/client/HttpClient.java

```diff
@@ -139,6 +139,52 @@ static HttpClient of(ClientFactory factory, URI uri, ClientOptions options) {
         return new HttpClientBuilder(uri).factory(factory).options(options).build();
     }
 
+    /**
+     * Creates a new HTTP client that connects to the specified {@link Endpoint} using the default
+     * {@link ClientFactory}.
+     *
+     * @param endpoint the URI of the server endpoint {@link Endpoint}
```

Comment:
`endpoint` is redundant.
`the server endpoint {@link Endpoint}` -> `the server {@link Endpoint}`